index.html

```diff
@@ -57,28 +57,13 @@ <h3 class="display-5">OpenFloodAI-Android</h3>
             </div>
         </section>
 
-        <section class="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center" id="connect">
-
-
-            <div class="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center moveleft">
-
-
-                <a class="twitter-timeline" data-width="600" data-height="400"
-                    href="https://twitter.com/OpenFloodAI?ref_src=twsrc%5Etfw">Tweets by OpenFloodAI</a>
-                <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-            </div>
-
-            <br><br><br>

```

Comment:
Hi @Fadhilamadan Can you please make add the Twitter feed again make PR?

Reply:
I have also made few changes in the repo!

Reply:
Already updated.

<img width="1440" alt="Screen Shot 2019-10-06 at 08 10 25" src="https://user-images.githubusercontent.com/19664424/66262749-27b87980-e811-11e9-999d-dfa3b4f70a42.png">